### ISSUE TITLE
Project License

### ISSUE THREAD

**Issue by ch98-1** [walterschell/Lua#31]
What is the license for this build script/project? Thanks,

**Comment by Particlebox** [comment]
Yes, please add a license. Without a license basically no one is allowed to use it: http://opensource.stackexchange.com/q/1720/775
